src/memex/views.py

```diff
@@ -82,7 +99,41 @@ def api_config(**settings):
         request_method = ('DELETE', 'GET', 'HEAD', 'POST', 'PUT',)
     settings['request_method'] = request_method + ('OPTIONS',)
 
-    return view_config(**settings)
+    if link_name:
+        link = {'name': link_name,
+                'method': initial_request_method,
```

Comment:
> I'm not sure if this works.

It does work for our current API methods. I've tested it :) However, the client doesn't support a list of methods so perhaps this should return just the _preferred_ method?